build/makelib/golang.mk

```diff
@@ -156,6 +156,12 @@ go.test: go.vet go.fmt $(CGO_PREREQS)
 #	@$(GOHOST) test -v -i $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 	@$(GOHOST) test -cover $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 
+.PHONY: go.lint
+go.lint: $(GO_TOOLS_DIR)/golint
+	@lint=`$(GO_TOOLS_DIR)/golint $(GO_ALL_PACKAGES)`; \
```

Comment:
The issue is that golint never returns a non-zero code. https://github.com/golang/lint/issues/65. So the code will always be 0 even if there are style errors.